### PR TITLE
The feature XXXX has been implemented

### PR DESCRIPTION
**Pull request by carlesaraguz**
- Change 1
- Fix A
- Bugfix D

### CODE REVIEW

File: AndreuSoler.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+New file created by Andreu Soler
```
Comment (carlesaraguz) [inline AndreuSoler.txt:1]: This is not correct, please fix.

Comment (joelortiz98) [inline AndreuSoler.txt:1]: Nice!

Comment (antonuxx) [inline AndreuSoler.txt:1]: Reviewed

File: AntonVillalonga [hunk 2]
```diff
@@ -0,0 +1 @@
+Anton's file, anton modification
```
Comment (andreusoler8) [inline AntonVillalonga:1]: Review it please

**Pull request merged**